frontend/keyboardNav.js

```diff
@@ -43,6 +43,17 @@ module.exports = function keyboardNav(store: Store, win: Object): (e: DOMEvent)
       return;
     }
     e.preventDefault();
+    if ((e.altKey && direction === 'right') || (e.ctrlKey && e.altKey && direction === 'right')) {
```

Comment:
`if (e.altKey && direction === 'right')` is enough to match both Alt+Right and Ctrl+Alt+Right, isn't it? And it wouldn't match Ctrl+Right.